packages/docusaurus-theme-classic/src/theme/DocSidebar/styles.module.css

```diff
@@ -12,6 +12,12 @@
     padding: 0.5rem;
     position: sticky;
     top: var(--ifm-navbar-height);
+    flex: 0 0 300px;
```

Comment:
If we want to have this here, we should extract this value out as css variables to be reused.